player/lua/stats.lua

```diff
@@ -930,13 +930,13 @@ end
 -- max_width      : (optional) limit line width to that many characters
 -- unlimited_lines: (optional) don't limit output lines and ignore scrolling
 local function compose_page(header, content, offset, max_width, unlimited_lines)
+    local terminal_size = mp.get_property_native("term-size")
```

Comment:
The property might not work on all platforms. This needs an option to allow override the automatic truncation.

Maybe even if it does work the user wants to never truncate, etc.